src/components/Post/Post.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
```

Comment:
Unless using the React object for lazy, suspense or memo etc, remove its import as its serving no purpose right now.